### PR TITLE
Add resume-after-hibernation service

### PR DESCRIPTION
**Pull request by agura-lex**
**Description**
Add systemd service to run asus-fan-control on resume from hibernate, it should be pulled as a dependency for the `asus-fan-control.service`

**Additional context**
Addresses https://github.com/dominiksalvet/asus-fan-control/issues/13

### CODE REVIEW

File: .gitpack/data/asus-fan-control.service [hunk 1]
```diff
@@ -12,3 +12,4 @@ ExecStart=/usr/local/bin/asus-fan-control
 
 [Install]
 WantedBy=multi-user.target
+Also=asus-fan-control-resume.service
```
Comment (dominiksalvet) [inline .gitpack/data/asus-fan-control.service]: I trust you with this as I do not know the syntax and the real importance of it... :smile:

Comment (agura-lex) [reply .gitpack/data/asus-fan-control.service]: It enables `asus-fan-control-resume` when `asus-fan-control` is getting enabled
https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Also=

Comment (agura-lex) [reply .gitpack/data/asus-fan-control.service]: Like when you run `systemctl enable asus-fan-control` it also does what `systemctl enable asus-fan-control-resume` would do.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 The changes not yet present in any release are listed in this section.
 
+### Fixed
```
Comment (dominiksalvet) [inline CHANGELOG.md]: Changed 'Added' to 'Fixed' as I consider this a bug fix, not a new feature.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -2,4 +2,4 @@
 * @dominiksalvet
 
 # systemd, AUR package
-.gitpack/data/asus-fan-control.service @agura-lex
+.gitpack/data/*.service @agura-lex
```
Comment (dominiksalvet) [inline .github/CODEOWNERS]: I have changed the ownership of systemd services to be more general.

File: .gitpack/data/asus-fan-control.service (hunk 1 above)
Comment (dominiksalvet) [reply .gitpack/data/asus-fan-control.service]: Great! :rocket:

**Comment by agura-lex** [comment]
Yup, LGTM as well!

**Comment by dominiksalvet** [comment]
Merged! Thank you for the contribution! :rocket:

**Pull request merged**